DocumentsApi/V1/Boundary/Response/GetClaimAndPreSignedDownloadUrlResponse.cs

```diff
@@ -1,8 +1,11 @@
 using System;
+using System.Diagnostics.CodeAnalysis;
 
 namespace DocumentsApi.V1.Boundary.Response
 {
-    public class ClaimAndDocumentResponse
+    // Suppress CA1055 because GetPreSignedURL returns a string, not an Uri
+    [SuppressMessage("ReSharper", "CA1056")]
+    public class GetClaimAndPreSignedDownloadUrlResponse
```

Comment:
This name sounds like a use case name. I think removing the verb in the name would solve that issue
```suggestion
    public class ClaimAndPreSignedDownloadUrlResponse
```

Reply:
Good spot! 👍